introduction.md

```diff
@@ -0,0 +1,6 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.